client-v2/src/bundles/frontsUIBundle.ts

```diff
@@ -213,12 +256,43 @@ const clearArticleFragmentSelection = (state: State, frontId: string) => ({
 
 const reducer = (state: State = defaultState, action: Action): State => {
   switch (action.type) {
+    case EDITOR_OPEN_CURRENT_FRONTS_MENU: {
```

Comment:
We had to lift this state out of React, as the fronts components need to know about it in order to get out of the way of our lovely menu